src/mlpack/methods/neighbor_search/neighbor_search.hpp

```diff
@@ -239,7 +238,7 @@ class NeighborSearch
    * Note that if you are calling Search() multiple times with a single query
    * tree, you need to reset the bounds in the statistic of each query node,
    * otherwise the result may be wrong!  You can do this by calling
-   * TreeType::Stat()::Reset() on each node in the query tree.
+   * "TreeType::Stat()::Reset()" on each node in the query tree.
```

Comment:
It was trying and failing to find a ref here.

Reply:
Does it work if you do `\c TreeType::Stat().Reset()` instead?  (It would be nice to get that in the code format, if it works.  If not, don't worry about it.)

Reply:
Good call!